src/html5/parser.rs

```diff
@@ -2817,6 +2805,9 @@ impl<'stream> Html5Parser<'stream> {
                 self.parse_error("doctype not allowed in before head insertion mode");
                 // ignore token
             }
+            Token::StartTagToken { name, .. } if name == "html" => {
```

Comment:
does this fix a test or did you just happen to find some unimplemented start-tag?

Reply:
in testing find this issue, and then reading docs, found `html` start-tag not implemented.